Packs/RedLock/Integrations/RedLock/README.md

```diff
@@ -1,799 +1,419 @@
+>| alert.status | dismissed,snoozed,open,resolved | true |
+>| alertRule.name |  | false |
+>| policy.subtype | build,run | true |
+>| resource.type |  | false |
+>| policy.complianceStandard |  | false |
+>| cloud.accountId |  | false |
+>| policy.severity | high,medium,low | true |
+>| policy.rule.type | cft,k8s,tf | true |
+>| cloud.service |  | false |
+>| policy.complianceRequirement |  | false |
+
+
+### redlock-get-remediation-details
+***
+Get remediation details for a given alert
+
+
+#### Base Command
+
+`redlock-get-remediation-details`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| alert-id | The alert id to get remediation details for | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Redlock.Alert.Remediation.Description | string | Description of CLI remediation instructions | 
+| Redlock.Alert.ID | string | The ID of the alert for which the remediation details applies | 
+| Redlock.Alert.Remediation.CLI | string | Exact CLI command string | 
+
+
+#### Command Example
+```!redlock-get-remediation-details alert-id=P-214016```
+
+#### Context Example
+```
+{
+    "Redlock": {
+        "Alert": {
+            "ID": "P-214016",
+            "Remediation": {
+                "CLI": "aws --region us-west-2 ec2 revoke-security-group-ingress --group-id sg-984392384bkhjb --ip-permissions '[{\"IpProtocol\": \"tcp\", \"IpRanges\":[{\"CidrIp\": \"0.0.0.0/0\"}]}]' ; aws --region us-west-1 ec2 authorize-security-group-ingress --group-id sg-98237498798 --ip-permissions '[{\"IpProtocol\": \"tcp\", \"FromPort\": 22, \"ToPort\": 22, \"IpRanges\":[{\"CidrIp\": \"10.0.0.0/8\", \"Description\": \"Enforced by Redlock Remediation\"}]}]'",
+                "Description": "\"This CLI command requires 'ec2:RevokeSecurityGroupIngress' permission. Successful execution will update the security group to revoke the ingress rule records open to internet either on IPv4 or on IPv6 protocol.\"} To resolve the alert from Prisma Cloud's console, add the permission."
+            }
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### remediationCLIoutput
+>|ID|Remediation|
+>|---|---|
+>| P-214016 | CLI: aws --region us-west-2 ec2 revoke-security-group-ingress --group-id sg-98234234 --ip-permissions '[{"IpProtocol": "tcp", "IpRanges":[{"CidrIp": "0.0.0.0/0"}]}]' ; aws --region us-west-2 ec2 authorize-security-group-ingress --group-id sg-92834324234h --ip-permissions '[{"IpProtocol": "tcp", "FromPort": 22, "ToPort": 22, "IpRanges":[{"CidrIp": "10.0.0.0/8", "Description": "Enforced by Redlock Remediation"}]}]'<br/>Description: "This CLI command requires 'ec2:RevokeSecurityGroupIngress' permission. Successful execution will update the security group to revoke the ingress rule records open to internet either on IPv4 or on IPv6 protocol."} To resolve the alert from Prisma Cloud's console, add the permission. |
```

Comment:
This table is not so human friendly. We can make it look better by changing the code a bit in the integration